packages/ui/src/components/va-time-picker/components/VaTimePickerColumn/VaTimePickerColumn.vue

```diff
@@ -182,5 +183,15 @@ export default defineComponent({
         background: var(--va-time-picker-cell-background-color-hover);
       }
     }
+
+    &:focus {
+      .va-time-picker-cell {
+        &--active {
+          @include focus-outline(4px);
```

Comment:
```suggestion
          @include focus-outline-inner(4px);
```

We can use it later in sidebar.